api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -202,6 +203,9 @@ message HttpConnectionManager {
 
     // Whether unix socket addresses should be considered internal.
     bool unix_sockets = 1;
+
+    // List of CIDR ranges that are treated as internal.
```

Comment:
Probably clarify here, if the `cidr_range` isn't specified, then only RFC1918 IP addresses will be considered internal